speech/noxfile_config.py

```diff
@@ -22,7 +22,7 @@
 
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7", "3.7", "3.9", "3.10", "3.11"],
+    "ignored_versions": ["2.7", "3.7", "3.9", "3.10", "3.11", "3.12"],
```

Comment:
The build issue relates to the wheel trying to be built for 3.12. I thought this config should allow this check to be non-blocking?

Reply:
we fixed it; the noxfile needs to be in the same level folder as the sample itself.